features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin_new/source/src/app/components/AdminPages/Addons/InputListBase.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { createMuiTheme } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Box from '@material-ui/core/Box';
+
+let id;
```

Comment:
Why we need to define a variable here ?

Reply:
Fixed with commit: https://github.com/wso2/carbon-apimgt/pull/8516/commits/c285a00fc8cc9c25a2bd9a0cc4fcc97187a91e15